client/src/components/Dashboard.tsx

```diff
@@ -139,13 +141,27 @@ const Dashboard = () => {
     }
   }
 
-  function renderSchedule(schedule: Schedule, viewMode: string) {
+  function sessionIsOver(end: string): boolean {
+    const zone = config?.timezone;
+    const endTime = parseTime(end, zone);
+    const now = DateTime.local();
+    return now >= endTime;
+  }
+
+  function renderSchedule(schedule: Schedule, viewMode: string, hideEndedSessions?: boolean) {
     const { rooms } = search(searchText, officeState.office, meetingsIndexed);
     const roomsIndexed = keyBy(rooms, (room) => room.roomId);
 
     const groupsWithRooms = selectGroupsWithRooms(meetingsIndexed, searchText, officeState.office);
     const groupsWithRoomsIndexed = keyBy<GroupWithRooms>(groupsWithRooms, ({ group }) => group.id);
 
+    if (hideEndedSessions) {
+      schedule = {
+        tracks: schedule.tracks,
+        sessions: schedule.sessions.filter((session) => !sessionIsOver(session.end)),
+      };
```

Comment:
It's unnecessary to copy the whole schedule if you only want to edit the sessions
```suggestion
      schedule.sessions = schedule.sessions.filter((session) => !sessionIsOver(session));
```